event-svc/src/tests/mod.rs

```diff
@@ -61,15 +61,9 @@ pub(crate) struct TestEventInfo {
 }
 
 async fn build_event_fixed_model(model: StreamId) -> TestEventInfo {
-    let controller = thread_rng()
-        .sample_iter(&rand::distributions::Alphanumeric)
-        .take(32)
-        .map(char::from)
-        .collect::<String>();
-
     let unique = gen_rand_bytes::<12>();
     let init = ceramic_event::unvalidated::Builder::init()
-        .with_controller(controller)
+        .with_controller(CONTROLLER.to_owned())
```

Comment:
tests were failing because a random controller isn't valid and events were no longer valid

Reply:
This is why the conclusion feed test expects were regenerated as well